### PR TITLE
Feature/activation date

### PR DESCRIPTION
**Pull request by johantjellden**
Part two, we now add support for StartDateUpdated events.

### CODE REVIEW

File: src/main/java/com/hedvig/notificationService/customerio/events/CustomerioEventCreatorImpl.kt [hunk 1]
```diff
@@ -67,4 +88,47 @@ class CustomerioEventCreatorImpl(private val productPricingFacade: ProductPricin
...
+
+        val returnMap = mutableMapOf<String, Any?>(
+            "name" to "ActivationDateUpdatedEvent"
+        )
+
+        val data = mutableMapOf<String, Any?>()
+        returnMap["data"] = data
+
+        val contractsWithStartDate = mutableListOf<MutableMap<String, Any?>>()
+        data["contractsWithStartDate"] = contractsWithStartDate
```
Comment (meletios) [inline src/main/java/com/hedvig/notificationService/customerio/events/CustomerioEventCreatorImpl.kt:108]: Shall we have all the hard coded values as constants?

File: src/main/java/com/hedvig/notificationService/customerio/events/CustomerioEventCreatorImpl.kt [hunk 2]
```diff
@@ -67,4 +88,47 @@ class CustomerioEventCreatorImpl(private val productPricingFacade: ProductPricin
...
+
+        val contractsWithoutStartDate = mutableListOf<MutableMap<String, Any?>>()
+        data["contractsWithoutStartDate"] = contractsWithoutStartDate
+
+        contracts.forEach {
+            if (it.startDate != null) {
+                contractsWithStartDate.add(
+                    mutableMapOf(
+                        "type" to if (it.type == AgreementType.NorwegianTravel) "reise" else "innbo",
```
Comment (meletios) [inline src/main/java/com/hedvig/notificationService/customerio/events/CustomerioEventCreatorImpl.kt:117]: Do we need to care about youth or not? - Its more a business related question

File: src/main/java/com/hedvig/notificationService/customerio/events/CustomerioEventCreatorImpl.kt (hunk 1 above)
Comment (johantjellden) [reply src/main/java/com/hedvig/notificationService/customerio/events/CustomerioEventCreatorImpl.kt:108]: I will convert this to proper data classes in next feature.

File: src/main/java/com/hedvig/notificationService/customerio/events/CustomerioEventCreatorImpl.kt (hunk 2 above)
Comment (johantjellden) [reply src/main/java/com/hedvig/notificationService/customerio/events/CustomerioEventCreatorImpl.kt:117]: Good question i will ask Sara :)

**Pull request merged**